website/routes.py

```diff
@@ -324,6 +332,16 @@ def make_url_map(app):
             json_renderer,
         ),
 
+        Rule(
+            [
+                '/project/<pid>/citation/<style>/',
+                '/project/<pid>/node/<nid>/citation/<style>/',
```

Comment:
A node only has one citation - which is basically metadata. I considered `/project/<pid>/citation/?style=<style>` as well, but decided this was better since we make limited use of URL parameters elsewhere.

In the near future, citations (plural) linked to a node might take the `<node path>/citations/` namespace for other things, too.